assignments/prototypes.js

```diff
@@ -15,13 +15,33 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
+function GameObject(atti){
+  Humanoid.call(this, atti);
+}
+
+CharacterStats.prototype = Object.create(GameObject.prototype);
```

Comment:
You are overwriting the the `CharacterStats` prototype here with that of the game object. Again this is the parent, you do not need to create or copy a prototype here. The prototype is just a list of functions that the object has access to(ability to use). 

You will need to do this when writing a child so that it has the same functions as the parent, but you can add new one the parent doesn't get.